packages/runtime-common/module-syntax.ts

```diff
@@ -451,14 +451,25 @@ function getProgramPath(path: NodePath<any>): NodePath<t.Program> {
 }
 
 function suggestedCardName(ref: { name: string; module: string }): string {
-  if (ref.name.toLowerCase().endsWith('card')) {
-    return ref.name;
-  }
   let name = ref.name;
   if (name === 'default') {
     name = ref.module.split('/').pop()!;
   }
-  return upperFirst(camelCase(`${name} card`));
+  if (
+    ref.module.startsWith('https://') &&
+    baseRealm.inRealm(new URL(ref.module))
+  ) {
+    // we suffix these specifically because we know they are fields and don't collide
+    // JS keywords like `String`, `Number`, and `Date`
+    name = `${name} field`;
+  }
```

Comment:
Does this handle the case for adding GeneralCard or GeneralField types? I think when adding a FieldDef field it's imported as `FieldDef as FieldDefCard`. There's also the quirky case after editing a field. When string field is edited, import turns into `StringField as StringFieldCard`.

Reply:
yeah the old functionality always slapped a `Card` suffix on anything that didn't already end in `Card`. this no longer does that

Reply:
tried again after clearing cache and looks good. it does say `StringField as StringFieldField` but not a big deal lol